src/app/teamSheets/_TeamSheetsBlankState/index.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+
+import clsx from 'clsx'
+
+import useIsMounted from '@hooks/useIsMounted'
+
+const TeamSheetBlankState = () =>  {
+
+  const isMounted = useIsMounted()
+  const isMobile = isMounted && window.innerWidth < 500
+  
+  return (
+    <div className={clsx(
+      "flex w-full mt-5",
+      isMobile ? "flex-wrap justify-center" : 'justify-between',
```

Comment:
i don't think `flex-wrap` does anything here, there is only 1 child

we also don't need `justify-between`, we can just apply `justify-center` conditionally

```
<div className={clsx(
  "flex w-full mt-5",
  { "justify-center": window.innerWidth < 500 },
)}>
```

Reply:
also we have the same problem here as i commented on for the TeamSheetCreateButton

the text starts on the left side and moves across once `isMobile` is true, i think we should have the same solution of hiding everything until it's loaded

```
if (!isMounted) return null
```

Reply:
Yeh sorry I have been a bit lazy cleaning up the code

I had been working on a blank state with an image on the right hand side which wrapped below on mobile but didnt like it so removed but forgot to cleanup all the code.

<img width="340" alt="Screenshot 2024-04-06 at 3 39 16 pm" src="https://github.com/dcarg/teamsheet/assets/39669538/2a18f012-8000-459f-be08-b8facadbd261">